spec/02-integration/02-cmd/09-prepare_spec.lua

```diff
@@ -65,6 +65,18 @@ describe("kong prepare", function()
     assert.truthy(helpers.path.exists(admin_error_log_path))
   end)
 
+  it("prepares a directory for LMDB", function()
+    assert(helpers.kong_exec("prepare -c " .. helpers.test_conf_path ..
+                             " -p " .. TEST_PREFIX))
+    assert.truthy(helpers.path.exists(TEST_PREFIX))
+
+    local lmdb_data_path = helpers.path.join(TEST_PREFIX, "dbless.lmdb/data.mdb")
+    local lmdb_lock_path = helpers.path.join(TEST_PREFIX, "dbless.lmdb/lock.mdb")
+
+    assert.truthy(helpers.path.exists(lmdb_data_path))
```

Comment:
Should check for the correct permission as well.

Reply:
done